crates/prql-compiler/src/utils/mod.rs

```diff
@@ -101,7 +101,7 @@ where
 
     fn any_true(self) -> Option<bool> {
         self.cloned()
-            .fold(Some(true), |a, x| a.zip(x).map(|(a, b)| a && b))
+            .fold(Some(true), |a, x| a.zip(x).map(|(a, b)| a || b))
```

Comment:
Yes this is backwards.

And it should also start with `Some(false)`.

Reply:
Also this comment is a bit confusing: https://github.com/PRQL/prql/pull/3405/files#diff-e59d2f5d0883e7d1ee71dfa13faef8218e2ef13dab795774895f4573cc86ad69R87-R89

If this is not used anywhere, maybe it should be removed because there are a few different interpretations of what it is supposed to do.

Reply:
OK, replacing this with #3413